app/models/manageiq/providers/azure/cloud_manager/vm_or_template_shared/scanning.rb

```diff
@@ -56,4 +56,18 @@ def has_proxy?
   def requires_storage_for_scan?
     false
   end
+
+  def require_snapshot_for_scan?
```

Comment:
Oh yes I misunderstood your original question.  Yes - when I add the other code for non-managed disk snapshots - this method will simply return true all the time.